packages/applications-service-api/src/controllers/submissions.js

```diff
@@ -1,47 +1,76 @@
+const md5 = require('md5');
 const Path = require('path');
 const { StatusCodes } = require('http-status-codes');
+const { createSubmission, updateSubmission } = require('../services/submission.service');
+const { uploadFile } = require('../services/ni.api.service');
 
 module.exports = {
 	async createSubmission(req, res) {
-		const { caseRef } = req.params;
-
-		const submissionId = parseInteger(req.body.submissionId) || Math.floor(Math.random() * 100);
-		const sequenceNumber = 1;
-
-		const responseBody = {
-			name: req.body.name,
-			email: req.body.email,
-			interestedParty: parseBoolean(req.body.interestedParty),
-			ipReference: req.body.ipReference,
-			deadline: req.body.deadline,
-			submissionType: req.body.submissionType,
-			sensitiveData: parseBoolean(req.body.sensitiveData),
-			lateSubmission: parseBoolean(req.body.lateSubmission),
-			submissionId: submissionId,
-			caseReference: caseRef,
-			dateSubmitted: new Date()
-		};
-
-		if (req.body.representation) {
-			responseBody.representation = req.body.representation;
-		} else if (req.file) {
-			responseBody.file = {
-				name: buildFileName(req.file.originalname, submissionId, sequenceNumber),
-				originalName: req.file.originalname,
-				size: req.file.size,
-				md5: 'dummymd5'
-			};
-		} else {
+		if (!req.body.representation && !req.file) {
 			return res.status(StatusCodes.BAD_REQUEST).send({
 				code: StatusCodes.BAD_REQUEST,
 				errors: ["Missing property 'representation' or 'file'"]
 			});
 		}
 
-		return res.status(StatusCodes.CREATED).send(responseBody);
+		const { caseReference } = req.params;
```

Comment:
It looks like a comment and file would be supported with one entry in the DB for both.  That may be fine but something to think about in the future ticket where we will need to create a PDF of comment and I think have a separate DB entry.  Think this is ok for now though.

Reply:
Looks like we already have ASB-801. I've added a couple of scenarios to try and distinguish the behaviour when submitting both a file and a comment